testutil/test_helper.go

```diff
@@ -15,6 +15,8 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+const DefaultRetryTimeout = 2 * time.Minute
```

Comment:
since `RetryFor` is always called with this default timeout, consider removing it as a parameter until we need it.

Reply:
Agreed, I will make that change in the next commit.